cpu/sam0_common/periph/flashpage.c

```diff
@@ -74,6 +84,11 @@ static void _lock(void)
 #else
     PAC1->WPSET.reg = PAC1_WPROT_DEFAULT_VAL;
 #endif
+
+    /* invalidate cache */
+#ifdef CMCC
+    CMCC->MAINT0.bit.INVALL = 1;
+#endif
```

Comment:
Why is this done?

Reply:
So we don't read old data from the cache that has been changed on the flash.